std/concurrency.d

```diff
@@ -2322,7 +2322,7 @@ private
         {
             assert( m_count );
             Node* n = r.m_prev;
-            enforce( n && n.next );
+            enforce( n && n.next, "attempting to remove invalid list node" );
```

Comment:
Not totally sure about these ones... but these look like they should be asserts anyway (these are inside methods inside a private data structure). @complexmath ?